src/main/java/core/basesyntax/machines/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.machines;
+
+public class Bulldozer extends Machine {
+

```

Comment:
And here as well. Please, review all files and fix it.

Reply:
Yeah, I already fixed it. Thanks for your participation!